sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Microsoft.Azure.Management.Blueprint.csproj

```diff
@@ -6,16 +6,17 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Blueprint</PackageId>
     <Description>Provides Microsoft Azure Blueprint management operations including the ability to create, update and delete Blueprint definition and assignment.</Description>
-    <Version>0.20.0-preview</Version>
+    <Version>0.20.5-preview</Version>
     <!-- Remove the Preview description in PackageReleaseNotes once the preview is removed.-->
     <AssemblyTitle>Microsoft Azure Blueprint Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Blueprint</AssemblyName>
     <PackageTags>Blueprint;</PackageTags>
     <PackageReleaseNotes>
 <![CDATA[This is a preview release sdk for Azure Blueprints, allows you to do Blueprint definition and assignment CRUD.
 
-Changes in 0.20.0-preview:
- - This version has breaking changes in ParameterValue model used in artifacts and blueprint assignment parameters.
+Changes in 0.20.5-preview:
```

Comment:
You can just remove this line -- the way nuget.org displays release notes is clear enough, and it can save some duplicated work when updating versions.

Reply:
Done